py-polars/polars/internals/lazy_functions.py

```diff
@@ -380,13 +380,84 @@ def sum(column: pli.Series) -> int | float:
 
 def sum(column: str | list[pli.Expr | str] | pli.Series | pli.Expr) -> pli.Expr | Any:
     """
-    Get the sum value.
+    Sum values in a column/Series, or horizontally across list of columns/expressions.
 
+    `pl.sum(str)` is syntactic sugar for:
```

Comment:
I believe you need double backticks to render code nicely in reST:

```suggestion
    ``pl.sum(str)`` is syntactic sugar for:
```

Also, makes more sense to me to include these in the Examples section below; but maybe that's just me.

Reply:
And thanks for the PR! We can definitely use better doc examples in many places 👍

Reply:
Thanks for the tip! I was just following along with the other docstrings in the code. I will look up the reST syntax.
My thinking for not including these lines in the Examples section was that they are explanatory information, rather than executable examples.
I'll work on the some of the other analogous functions in a separate PR I think.